lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -245,7 +239,7 @@ export class UserAgentApplication {
    * @param {errorReceivedCallback} errorCallback - Callback which contains a AuthError object, containing error data from either the server
    * or the library, depending on the origin of the error.
    */
-  setRedirectCallbacks(successCallback: tokenReceivedCallback, errorCallback: errorReceivedCallback): void {
+  handleRedirectCallbacks(successCallback: tokenReceivedCallback, errorCallback: errorReceivedCallback): void {
```

Comment:
setRedirectHandlers ?

Reply:
much better name. +1

Reply:
we are handling auth response, but here we are setting the handler right ? not actually handling, it makes it clearer we are setting something that will handle at some future point imo

Reply:
Line 257-272 also handles the callback. That is why I changed it from set before

Reply:
just a naming opinion, just want to make sure in the public api, it clearly and concisely conveys what the function is doing.  If everyone feels good about that, I am fine with it

Reply:
Resolve for now, we can revisit later if needed